Dockerfile

```diff
@@ -41,6 +41,10 @@ RUN set -x \
   # Install cypress
   && cypress install --cypress-version ${CYPRESS_VERSION} \
 
+  # Install dcos-launch
+  && curl 'https://downloads.dcos.io/dcos-test-utils/bin/linux/dcos-launch' > /usr/local/bin/dcos-launch \
```

Comment:
Suggestion: Instead of using as `&&` we would benefit from Docker caching if we use multiple `RUN` that way docker will create a layer and re-use if possible.

Reply:
Yeah, I believe that @wavesoft decided to use only one `RUN` call to reduce the number of mounts which otherwise could potentially lead to issues. I think it should be fine in our case  though.  Anyway, let's not refactor our docker setup in this PR. 

Could you please file a JIRA to revisit our setup?

Reply:
There are pros and cons to this. On the one hand we leverage the layer fs, on the other hand the resulting image size is bigger. In addition to that it usually makes sense to have multiple layers only when you expect frequent changes so you leverage caching and incremental updates.

Reply:
FYI: created a jira to address empty continuation line warning https://jira.mesosphere.com/browse/DCOS-19026

Reply:
Great feedback, thanks.